cmake/ElektraCache.cmake

```diff
@@ -277,6 +277,14 @@ option (INSTALL_SYSTEM_FILES "Install files to system directories" ON)
 
 option (INSTALL_BUILD_TOOLS "Install build tools for cross-compilation" OFF)
 
+option (HASHLOOKUP "Add the possibility to lookup with an Order Preserving Minimal Perfect Hash Map" ON)
```

Comment:
I would prefer a more generic option and a name which is more consistent with other options.

What about `ENABLE_OPTIMIZATIONS` with the comment "Tturn on optimizations that trade memory for speed"?